modules/admin_manual/pages/configuration/files/manual_ file_ locking.adoc

```diff
@@ -0,0 +1,35 @@
+= Manual File Locking
+:toc: right
+
+== Introduction
+
+ownCloud Server 10.5 comes with great enhancements for content collaboration. Manual file locking allows users to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out).
+
+The feature builds on the WebDAV Locks backend which has been introduced with Server 10.1 and is now available in the ownCloud Web Interface. Using the context menu of files, every user who has access can lock them. Users can recognize locked files by the means of a new lock indicator. While a file is locked, other users can still access it but they can not make any changes. Locked files can manually be unlocked by the lock owner (the user who locked the file; exclusive locking) using the "Locks" tab in the file details view (right sidebar).
```

Comment:
> The feature builds on the WebDAV Locks backend which has been introduced with Server 10.1 and is now available in the ownCloud Web Interface.

This text is the technical background, but I doubt it gives valueable information. If I would read it, I would ask myself, is this for webdav based mounts only? Can I use it on SMB, GD, ect? I propose a text which describes which mount types are in/excluded from this feature. After clearing that, the next sentence could start with:
`Every user has access to lock files. You can find it using the files context menu`
[add a picture here]
`...other users can still access it but they can not make any changes...` --> `other users can still access it but they can not save any changes during the lock`

`Locked files can manually be unlocked by the lock owner (the user who locked the file; exclusive locking) using the "Locks" tab in the file details view (right sidebar).` --> `Locked files can manually be unlocked by the lock owner. The lock owner is the user who locked the file. This type of locking is called exclusive locking. To unlock such a file, use the "Locks" tab in the file details view (right sidebar).`

Reply:
Where can I find out which storages are supported? Or ist this a question for @pmaier1 ?

Reply:
Valid point, I cant tell.
@pmaier1 can you give your input pls?

Reply:
All storages are supported as locking takes place on the WebDAV level. But it's important to understand that the locks will only be available via ownCloud, not when a user works directly on the storage.